src/components/video.tsx

```diff
@@ -48,6 +48,8 @@ export const Video = (props: VideoProps) => {
     })()
   );
 
+  let [keyPressed, setKeyPressed] = useState('')
```

Comment:
We don't need this state, since the keypress event can detect if shift is pressed as well. Besides, this state is not reset as well at the moment - so, even if the shift key is released, this state remains set to "shift" so just pressing n/p will continue to work.

Reply:
Thanks for pointing it out,
I realized this late !
Hence I set back the state value to empty string after "N or P" was pressed. But still its not a good idea because the state holds even after shift is released
u were right, lets not use hooks for this problem